spec/features/create_object_h2_spec.rb

```diff
@@ -56,7 +56,10 @@
     fill_in 'Abstract', with: "An abstract for #{collection_title} logo"
     fill_in 'Keyword', with: 'Integration test'
 
-    check('I agree to the SDR Terms of Deposit')
+    # if you have previously agree to the terms within the last year, there will be no checkbox
+    if page.has_css?('#work_agree_to_terms', wait: 0)
```

Comment:
rubocop 👮‍♂️ 🚨 ❗

Reply:
🏫  went to traffic school after getting pulled over